packages/workers-shared/utils/helpers.ts

```diff
@@ -1,22 +1,15 @@
+import { isAbsolute, sep } from "node:path";
 import { getType } from "mime";
 
-/** normalises sep for windows, and encodes each segment */
-export const encodeFilePath = (filePath: string, sep: string) => {
-	const encodedPath = filePath
-		.split(sep)
-		.map((segment) => encodeURIComponent(segment))
-		.join("/");
+/** normalises sep for windows and prefix with `/` */
+export const normalizeFilePath = (relativeFilepath: string) => {
+	if (!isAbsolute(relativeFilepath)) {
```

Comment:
"if not isAbsolute, thow expected relative path error"

i.e.

"if is relative, throw expected relative path error"

is that backwards?

Reply:
😱  - good spot!

Reply:
yes!! very backwards! currently fixing and have uncovered more issues when there's a user worker 😱